SparkPost/2web2ui
# FE-1157 - [Dashboard UX Redesign] Incorporate artwork from Matchbox

[FE-1157](https://sparkpost.atlassian.net/browse/FE-1157)

### What Changed

- Adds new `<Picture/>` and `<Picture.Img />` components along with supporting tests and documentation
- Adds `url-loader` to Storybook webpack config to support image loading
- Adds support for `.webp` images via the `url-loader` Webpack plugin
- Adds new UI to the dashboard V2 page consisting of an image and some links

### How To Test

- On an account with the `allow_dashboard_v2` account flag, go to `/dashboardV2` and verify the illustration and links design in the mocks matches the app - **Note:** UX would prefer that the image is not visible on smaller screen sizes. A bug with the Matchbox Column component was discovered as a result and an issue was filed - https://github.com/SparkPost/matchbox/issues/625 - this can be addressed at a later date and time

### To Do
